packages/test/mocha-test-setup/api-report/mocha-test-setup.api.md

```diff
@@ -4,8 +4,6 @@
 
 ```ts
 
-/// <reference types="mocha" />
```

Comment:
Intentional change?

Reply:
I HATE this file. running build randomly adds/removes this line. It failed build because of it. Will fix.

Reply:
Fixed